api4/team.go

```diff
@@ -1200,8 +1200,11 @@ func inviteUsersToTeam(c *Context, w http.ResponseWriter, r *http.Request) {
 	if graceful {
 		cloudUserLimit := *c.App.Config().ExperimentalSettings.CloudUserLimit
 		var invitesOverLimit []*model.EmailInviteWithError
-		if cloudUserLimit > 0 && c.IsSystemAdmin() {
-			emailList, invitesOverLimit, _ = c.App.GetErrorListForEmailsOverLimit(emailList, cloudUserLimit)
+		if *c.App.Srv().License().Features.Cloud && cloudUserLimit > 0 && c.IsSystemAdmin() {
+			subscription, _ := c.App.Cloud().GetSubscription()
+			if subscription.IsPaidTier != "true" || subscription == nil {
```

Comment:
I'd check first if the subscription is nil because if it is we're going to get a panic
```suggestion
			if subscription == nil || subscription.IsPaidTier != "true" {
```